dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,136 @@
+# Lit release process
+
+Audience: Lit core contributors.
+
+## Objective
+
+The end result of this document is a release of all unreleased changes on the main branch for all packages in the Lit monorepo.
+
+> **Note**
+> Before releasing, ensure the main branch has been imported and tested within Google.
+
+### Tl;dr
+
+1. Approve "Release / Release" GitHub action on the latest commit to release.
+1. Code review the automatically created "Version Packages" PR.
+1. Squash and merge the "Version Packages" PR.
+1. Approve the "Release / Release" workflow on the commit created by "Version Packages" PR. This publishes the new releases to NPM and GitHub.
+
+## Process
+
+1. Find the [commit you want to release](https://github.com/lit/lit/commits/main), and click on the yellow dot (or red cross) that represents GitHub Action checks. There should be a pending Github Action called "Release / Release (push)". Click "Details".
+
+![GitHub Actions UI showing the pending Release action. A link has the text "Review pending deployments"](./images/lit-release-process/release-github-action-review-pending.png)
```

Comment:
Is this the right image? I don't see a "Review pending deployments" link in it

Reply:
This is not! Thank you so much!